artifacts/Project.toml

```diff
@@ -1,5 +1,6 @@
 [deps]
 ArtifactWrappers = "a14bc488-3040-4b00-9dc1-f6467924858a"
 ClimaAtmos = "b2c96348-7fb7-4fe0-8da9-78d88439e717"
+ClimaCoupler = "4ade58fe-a8da-486c-bd89-46df092ec0c7"
```

Comment:
Any reason why we need to add this?

Reply:
I think this comes from running the `up_deps.jl` script ? cc @AlexisRenchon

Reply:
I am not sure. 
I actually wasn't aware or `up_deps.jl` and was updating Manifests manually, but it is a bit tedious and it is a good idea to have a script. 

I don't see why we would need ClimaCoupler as a deps in artifacts. Should we ask Gabriele?

Reply:
Yes, `ClimaCoupler` should not be there. 

The artifacts project.toml depends on the other packages because we have to access their artifacts.

Reply:
@akshaysridhar can you remove ClimaCoupler deps and merge this PR? (or are we waiting for something else?)